configure.ac

```diff
@@ -970,6 +970,8 @@ if test "$TARGET_OS" = "darwin"; then
   AX_CHECK_LINK_FLAG([-Wl,-bind_at_load], [HARDENED_LDFLAGS="$HARDENED_LDFLAGS -Wl,-bind_at_load"], [], [$LDFLAG_WERROR])
 fi
 
+AX_CHECK_LINK_FLAG([-Wl,--build-id=0xfedcba9876543210], [LDFLAGS="$LDFLAGS -Wl,--build-id=0xfedcba9876543210"], [AC_MSG_ERROR([linker did not accept --build-id flag])], [$LDFLAG_WERROR])
```

Comment:
Of course, it should use git commit hash or some another tag, but not be hardcoded :)